src/Event/Productions/ProductionRepository.php

```diff
@@ -151,4 +151,33 @@ public function findProductionForEventId(string $eventId): Production
 
         return $production;
     }
+
+    /**
+     * @param string $forEventId
+     * @param ProductionId $inProductionId
+     *
+     * @return Tuple[]
+     * @throws EntityNotFoundException
+     */
+    public function findTuples(string $forEventId, ProductionId $inProductionId): array
```

Comment:
Similar to the remark here (https://github.com/cultuurnet/udb3-silex/pull/474/commits/6ff814af96371417e6be6c0d500cbc11fb095b2f#r449185330), I would rename this method to `findEventPairs()`

Reply:
We should also add a test for this repository method.

Reply:
Done (and I've caught a bug that I missed, testing FTW)